docs/podman-create.1.md

```diff
@@ -244,6 +244,16 @@ inside of the container.
 
 Read in a line delimited file of environment variables
 
+**--http-proxy**=*true*|*false*
+
+By default proxy environment variables are passed into the container if set
+for the podman process.  This can be disabled by setting the `--http-proxy`
+option to `false`.  The environment variables passed in include `http_proxy`,
+`https_proxy`, `ftp_proxy`, `no_proxy`, and also the upper case versions of
+those.
+
+Defaults to `true`
+
```

Comment:
It would be nice to have an example in the man pages showing these in use.

Reply:
fixed in #3047